packages/graphql/tests/integration/directives/cypher.int.test.ts

```diff
@@ -993,5 +994,109 @@ describe("cypher", () => {
                 );
             });
         });
+        // Reproduces https://github.com/neo4j/graphql/issues/630
```

Comment:
This is great! One point for me, please could you put this test in the `/issues` folder?

Reply:
Fixed in https://github.com/neo4j/graphql/pull/821/commits/82ecfac0ac87ab4505225d9c0ac09f5c62782323